packages/core/src/components/datetime/test/default/datetime.e2e.ts

```diff
@@ -0,0 +1,63 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/datetime/test/default/index.html';
+
+test.describe('tds-datetime-default', () => {
+  test('renders datetime component correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    /* Check diff on screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('verifies label, helper text, size, and calendar icon for datetime-local component', async ({
+    page,
+  }) => {
+    await page.goto(componentTestPath);
+
+    // Check for the label text
+    const label = page.locator('.tds-datetime-label');
+    const dateTime = page.locator('tds-datetime');
+    const helperText = page.locator('.tds-datetime-helper .tds-helper');
+    const calendarIcon = page.locator('tds-icon[name="calendar"]');
```

Comment:
The spreadsheet is asking for a check of these props/attributes too:

- min
- max
- disabled 
- no-min-width

Reply:
Yes, I saw that too, but only empty values, and when empty it is not rendered right?